lab-judy/lib/error-middleware.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const createError = require('http-errors');
+const debug = require('debug')('city:server');
```

Comment:
Your debug isn't working because you are specifying the debug should look for 'note*' in the modules it should read from, and here you are giving it 'city*'.